stock_picking_auto_create_lot/readme/CONFIGURE.md

```
@@ -1,6 +1,18 @@
-To configure this module, you need to:
+To use the auto-create feature you need to enable tracking by lots/serial numbers:
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)